test/misc.jl

```diff
@@ -722,3 +722,11 @@ end
        end
    end
 end
+
+@testset "remove_linenums!" begin
+    # Ensure line information is removed from macro calls
+    ex = :(@macro argument)
+    @test any(arg->arg isa LineNumberNode, ex.args)
+    Base.remove_linenums!(ex)
+    @test !any(arg->arg isa LineNumberNode, ex.args)
```

Comment:
```suggestion
    @test all(arg->if (arg isa LineNumberNode) arg == LineNumberNode(1, :none) end, ex.args)
```
If you make the above change, you'll need to change this too for your tests.